core/templates/base-components/base-content.directive.html

```diff
@@ -81,4 +88,26 @@
     transform: translateX(50%);
     width: 20px;
   }
+  .navbar-mobile-container {
+    bottom: 0;
+    display: block;
+    position: fixed;
+    right: 0;
+    text-align: right;
+    width: 100%;
+    z-index: 999;
+  }
+  .navbar-mobile-options {
+    background-color: #e6e6e6;
+    border-top: 1px solid #000000;
```

Comment:
Can we use #000? (Generally use 3 chars for repeated hex values)

Reply:
Done